src/main/java/com/microsoft/aad/msal4j/UserNamePasswordParameters.java

```diff
@@ -14,19 +14,30 @@
 /**
  * Object containing parameters for Username/Password flow. Can be used as parameter to
  * {@link PublicClientApplication#acquireToken(UserNamePasswordParameters)}
+ *
+ * For more details, see https://aka.ms/msal4j-username-password
```

Comment:
Can we update aka.ms link to https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-desktop-acquire-token?tabs=java#username-and-password